src/sections/featured-product.liquid

```diff
@@ -2,54 +2,91 @@
 {%- assign current_variant = product.selected_or_first_available_variant -%}
 {%- assign featured_image = current_variant.featured_image | default: product.featured_image -%}
 
-{% if product == empty %}
-  {%- assign section_onboarding = true -%}
-  {%- assign onboarding_title = 'homepage.onboarding.product_title' | t -%}
-{% endif %}
+{%- if product == empty -%}
+  {%- assign default_product_title = 'homepage.onboarding.product_title' | t -%}
+{%- endif -%}
+
+<section data-section-id="{{ section.id }}" data-section-type="product" class="featured-product">
+
+  <div class="featured-product__image-container">
+    {%- unless product.images == empty -%}
+      {%- for image in product.images -%}
+        {%- capture wrapper_class -%}
+          {%- unless featured_image == image -%}
+            hide
+          {%- endunless -%}
+        {%- endcapture -%}
+        {%- include 'responsive-image' with
+          image: image,
+          max_width: 700,
+          max_height: 800,
+          image_attributes: "data-product-featured-image",
+          wrapper_class: wrapper_class,
+          wrapper_attributes: "data-product-image-wrapper",
+        -%}
+      {%- endfor -%}
+      {%- if product.images.size > 1 -%}
+        <ul class="featured-product__image-thumbnails">
+          {%- for image in product.images -%}
+            <li>
+              <a href="{{ image.src | img_url: '480x480' }}" data-thumbnail-id="{{ image.id }}" data-product-single-thumbnail>
+                <img class="product-single__thumbnail-image" src="{{ image.src | img_url: '160x160' }}" alt="{{ image.alt | escape }}">
+              </a>
+            </li>
+          {%- endfor -%}
+        </ul>
+      {%- endif -%}
+    {%- else -%}
```

Comment:
We shouldn't have this `{% else %}` because not all products require images. If a merchant wants a product page without an image, we should make it possible.

Reply:
🥇 Good catch. Still leaving an empty div in that case to keep everything aligned in the grid.